docs/modules/data-structures/pages/backing-up-maps.adoc

```diff
@@ -0,0 +1,165 @@
+= Backing Up Maps
+:description: Backing up maps helps to protect your data in case a cluster member goes down. For each backup, Hazelcast creates a copy of the map and distributes it across another member. Because each backup is a copy, the number of backups in your cluster affects the amount of memory that your members need.
```

Comment:
```suggestion
:description: Backing up maps helps to protect your data in case a cluster member goes down. For each backup, Hazelcast creates a copy of the map entries and distributes them to another member. Because each backup is a copy, the number of backups in your cluster affects the amount of memory that your members need.
```